src/node/include/nodeclient.h

```diff
@@ -16,32 +16,36 @@
 #define _NODECLIENT_H_
 
 #include "croseclient.h"
+#include "srv_accept_reply.h"
 #include "srv_srv_select_reply.h"
+#include "srv_switch_server.h"
+
+#include "cli_accept_req.h"
+#include "cli_login_req.h"
+#include "cli_join_server_req.h"
 
 class NodeClient : public RoseCommon::CRoseClient {
  public:
   NodeClient();
   NodeClient(std::unique_ptr<Core::INetwork> _sock);
+  ~NodeClient();
```

Comment:
shouldn't the destructor be virtual?

Reply:
No it's not virtual because it's the final destructor. No one should be inheriting off of this